rules/falco_rules.yaml

```diff
@@ -490,6 +490,20 @@
     WARNING
   tags: [host, container, filesystem, mitre_discovery, T1546.004]
 
+- rule: Drop and execute new binary in container
+  desc:
+    Detect if an executable not belonging to the base image of a container is being executed.
+    The drop and execute pattern can be observed very often after an attacker gained an initial foothold.
+    is_exe_upper_layer filter field only applies for container runtimes that use overlayfs as union mount filesystem.
+  condition: spawned_process and container and proc.is_exe_upper_layer=true
```

Comment:
This is awesome!  @loresuso you should write a blog about this rule.

Reply:
Thank you @Kaizhe, really appreciated! 🙏 
Yes, I was thinking about it, I will definitely do it!